src/cli-validator/runValidator.js

```diff
@@ -230,13 +231,21 @@ const processInput = async function(program) {
       exitCode = 1;
       continue;
     }
-
+    if (errorsOnly) {
+      results.warning = false;
+    }
```

Comment:
This is non-obvious code as we discussed yesterday - it is good practice to add comments explaining why code exists if it isn't immediately clear. Will you add a comment above this block? Something like
> the `warning` property tells the user if warnings are included as part of the output
> if `errorsOnly` is `true`, only errors will be returned, so need to force this to `false`

Reply:
sorry this should've been something I thought of myself, I will make sure to do this in the future.